apps/foundation-demo/src/app/design/molecules/navbar/navbar/navbar.component.scss

```diff
@@ -0,0 +1,8 @@
+@import 'daffodil-util';
+
+.daff-navbar {
+  display: flex;
+  align-items: center;
+  padding: 8px 16px;
+  @include section-divider();
```

Comment:
Bit of a problem I encountered with these two lines (6,7). It's really difficult to get what Elain designed for the foundation-demo header to work with this padding and section-divider. The border-right of the menu button is supposed to go all the way down to this navbar border, but it can't because of the padding here. But then if you remove the padding, the border-bottom is messed up. And you also can't fix it, because header wraps daff-navbar. Do you want to just get rid of navbar?

Reply:
I still like navbar it can act as a `height` enforcer and a `layout` enforcer with display flex.